readme.md

```diff
@@ -95,11 +88,7 @@ Non-primitive data types are also known as reference data types. They are create
 - [Class](Data%20Types/readme.md#class)
 
 ## [Statements and Functions](#statement-and-function)
-In computer programming, a statement is a syntactic unit of an imperative programming language that expresses some action to be carried out. A program written in such a language is formed by a sequence of one or more statements. A statement may have internal components (e.g., expressions).
```

Comment:
Why are you removing this?

Reply:
How do I withdraw the pull request.  I don't have time today.  I'll need to start over.

Reply:
You can close the pull request.